app/js/lib/foxbox/foxbox.js

```diff
@@ -180,32 +149,29 @@ export default class Foxbox extends Service {
         // we have no record of previous registrations, we schedule
         // a retry.
         if (!this[p.boxes].length &&
-            !this[p.settings]._localOrigin &&
-            !this[p.settings]._tunnelOrigin) {
+            !this[p.settings].localOrigin &&
+            !this[p.settings].tunnelOrigin) {
           throw new Error('Registration service did not return any boxes.');
         }
 
         if (!this[p.settings].configured) {
           this.selectBox();
         }
-
-        this._dispatchEvent('box-online', true);
```

Comment:
Actually box is not online until we receive any ping response from local or remote box. It's just registration service online, so I've removed that.